lib/index.ts

```diff
@@ -34,7 +34,7 @@ type HapiOidcOptions = {
 
 const logger = (server: Server) => (
   tags: string | Array<string>,
-  message?: object | string,
+  message?: Record<string, unknown> | string,
```

Comment:
one of the default linter rules says that `object` sucks, and i think chester told me this once and I forgot about it. The linter annotation says `Record<string, unknown>` is better when you don't want to use `object` or `{ [key: any]: any }`.

I don't really know if this is good, but I did include it everywhere in this PR so I'm looking to get corrected on this if this is wrong.

Reply:
I don't haven't use the Record built-in, but after reading about it, it seems legit.